test/compilable/extra-files/testcolor.sh

```diff
@@ -0,0 +1,43 @@
+compare()
+{
+    local actual expected
+    actual="$1"
+    expected="$2"
+    if [ "$actual" != "$expected" ] ; then
+        echo "Expected: $expected"
+        echo "Actual:   $actual"
+        echo "$expected" | hexdump -C
+        echo "$actual" | hexdump -C
+        exit 1
+    fi
+}
+
+filterAnsi()
+{
+    cat | sed "s/$(printf "\x1b")\(\[[0-9;]*m\)/[\1/g" | sed "s/\[\[/[/g" | sed "s/m\[m/m/g" | tr -d "\n" | tr -d "\r"
```

Comment:
- useless cat
- last two `tr`s look like they can be concatenated

Reply:
Seems overly convoluted (why not just put the escapes directly in the expected string?) but OK

Reply:
Ask @wilzbach ;), but sure.